src/main/ruby/core/numeric_mirror.rb

```diff
@@ -47,15 +47,17 @@ def step_float_size(value, limit, step, asc)
         end
       end
 
-      def step_size(limit, step)
-        values = step_fetch_args(limit, step)
+      def step_size(limit, step, by)
```

Comment:
It sounds intentional, maybe to avoid an accidental use expecting limit/step to be reversed. See
https://github.com/ruby/ruby/commit/fd4b5b8dc83e61bbbb62fa7e6194c0ece6b9349f.
@knu Do you remember why the positional `step` argument is checked against zero but the keyword arg `by:` is not?

@havenwood In any case, it seems good to follow MRI here.